.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ on:
 jobs:
   test:
     name: Test
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
We decided in the latest PR about this to use the version here, to be sure in what version we build. In this way it will not fail when changing from one version to another without noticing.

Reply:
Got a `waiting for a runner to pickup this job` error using `ubuntu-22.04`. Might still work with ubuntu-20.04 in ci.yml 

Strange enough using `ubuntu-22.04` does work on the other workflows.

Reply:
According to this https://github.com/actions/virtual-environments/discussions/5427 we will need to wait some more days for Ubuntu 22.04.

Reply:
For now 20.04 works. Will update ci.yml later.